lua/default-config.lua

```diff
@@ -488,6 +488,28 @@ lvim.lang = {
       },
     },
   },
+  fortran = {
+    formatters = {},
+    linters = {},
+    lsp = {
+      provider = "fortls",
+      setup = {
+        cmd = {
+          DATA_PATH .. "/lspinstall/fortran/venv/bin/fortls",
+        },
+        on_attach = common_on_attach,
+        on_init = common_on_init,
+        capabilities = common_capabilities,
+        root_dir = function(fname)
+          local util = require "lspconfig/util"
+          return util.root_pattern(".fortls", ".git")(fname)
+        end,
+        settings = {
+          nthreads = 2,
```

Comment:
Out of curiosity, why setting it to 2 when the default is 1 ?

Reply:
Not thing in particular. The default of `fortls` is actually 4 threads, but the default of the `lspconfig` is 1 somehow. In practice I found 1 thread is usually quite slow for large codebase, but some (really) old machine may not have 4 threads so I opt for 2.